### DISCUSSION TITLE
Admin console access from a different local and remote operating system

### DISCUSSION THREAD

**Discussion by daringh10** [pocketbase/pocketbase#1858] [Q&A] [answered]
If I work on a Mac locally and want to access the admin dashboard UI on my debian hosted server, is there a way to do this? 

Would the only way to access the admin dashboard be to have the same OS on both local and remote and makes changes locally then copy the executable over to the server? 

Is there another way of going about this?

**Comment by arslanastral** [comment, marked as answer]
The admin dashboard is a web interface embedded inside pocketbase's go binary and served by it. You simply need to access the IP:PORT or domain of pocketbase on debian hosted server from your Mac and as long as the appropriate ports are open and firewall rules are not blocking, it should work.

_I should also mention, if you were asking for remote development then you should look into [SSH](https://www.digitalocean.com/community/tutorials/ssh-essentials-working-with-ssh-servers-clients-and-keys) and [vscode remote dev](https://code.visualstudio.com/docs/remote/remote-overview) (if you use it)_
